frontend/src/components/design-system/navigation/BottomBar.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+
+import ContainerComponent from '@/components/design-system/containers/ContainerComponent.vue';
+
+export interface Props {
+  desktop?: boolean;
+  mobile?: boolean;
+  transparent: 'always' | 'never' | 'desktop' | 'mobile';
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  desktop: true,
+  mobile: true,
+});
+
+const classes = computed(() => ({
+  'bottom-bar--transparent': props.transparent === 'always',
+  'bottom-bar--transparent-desktop': props.transparent === 'desktop',
+  'bottom-bar--transparent-mobile': props.transparent === 'mobile',
+  'bottom-bar--desktop': props.desktop,
+  'bottom-bar--mobile': props.mobile,
+}));
+</script>
+
+<template>
+  <ContainerComponent :class="['bottom-bar', classes]" padding="small">
+    <slot />
+  </ContainerComponent>
+</template>
+
+<style scoped lang="scss">
+.bottom-bar {
+  display: none;
+
+  &--desktop {
+    @media (min-width: 700px) {
+      display: flex;
```

Comment:
Není uveden příklad použití a hledat v návrhu se mi to nechce, ale je bezpečnější použít `display: block`, protože s flexem měníš layout komponenty. Za mě je lepší, pokud by se komponeta do slotu dávala již připravená a nespoléhala na to, že ji styly bude určovat tato komponenta.

Reply:
Může být. Opravuju.